test/lib/lbt/analyzer/XMLTemplateAnalyzer.js

```diff
@@ -3,7 +3,7 @@ const XMLTemplateAnalyzer = require("../../../../lib/lbt/analyzer/XMLTemplateAna
 const ModuleInfo = require("../../../../lib/lbt/resources/ModuleInfo");
 const sinon = require("sinon");
 
-test("integration: Analysis of an xml view", async (t) => {
+test("integration: Analysis of a xml view", async (t) => {
```

Comment:
an?

Reply:
AFAIK, "an" has to be used whenever the next following word starts with a vocal __when spoken__, not __when written__. And "XML" is spoken "EX M L". Much like "SAP" is spoken "ESS AY PEE". That's why we also write "an sap.m.Button".

Reply:
Actually didn't know that, thanks 👍